drivers/mtd_sdcard/mtd_sdcard.c

```diff
@@ -85,22 +85,6 @@ static int mtd_sdcard_read_page(mtd_dev_t *dev, void *buff, uint32_t page,
     return res * SD_HC_BLOCK_SIZE;
 }
 
-static int mtd_sdcard_write(mtd_dev_t *dev, const void *buff, uint32_t addr,
-                            uint32_t size)
-{
-    DEBUG("mtd_sdcard_write: addr:%" PRIu32 " size:%" PRIu32 "\n", addr, size);
-    mtd_sdcard_t *mtd_sd = (mtd_sdcard_t*)dev;
-    sd_rw_response_t err;
-    sdcard_spi_write_blocks(mtd_sd->sd_card, addr / SD_HC_BLOCK_SIZE,
-                            buff, SD_HC_BLOCK_SIZE,
-                            size / SD_HC_BLOCK_SIZE, &err);
-
-    if (err == SD_RW_OK) {
-        return 0;
-    }
-    return -EIO;
-}
```

Comment:
This function def needs to be removed since the writes of less than SD_HC_BLOCK_SIZE are not possible.